src/main/kotlin/org/web3j/evm/ConsoleDebugTracer.kt

```diff
@@ -75,7 +79,7 @@ open class ConsoleDebugTracer(protected val metaFile: File?, private val reader:
             metaFile,
             lastSourceFile,
             byteCodeContractMapping,
-            sourceFileBodyTransform
+            sourceFileBodyTransform,
```

Comment:
why we have this comma here?

Reply:
When I am applying spotless apply it is adding the comma there. 
Else giving error - 
`Execution failed for task ':spotlessKotlinCheck'.
> The following files had format violations:
      src/main/kotlin/org/web3j/evm/ConsoleDebugTracer.kt
          @@ -79,7 +79,7 @@
           ············metaFile,
           ············lastSourceFile,
           ············byteCodeContractMapping,
          -············sourceFileBodyTransform
          +············sourceFileBodyTransform,
           ········)
           ····}
`